src/content/en/2019/fonts.md

```diff
@@ -0,0 +1,118 @@
+
+**06.10: Italics**
+
+Only 29% of pages declare at least one italic font. This suggests a lot of faux-italic across the web, as web browsers will synthesize a fake bold version if one is not available for use. That means using elements with italic user agent styling (`em`, `i`, et al) will render faux-italic.
+
+Faux bold may be even larger of a problem, as about 53–55% of pages have a font-face with a weight above 400, and (arguably) bold usage is more common than italic usage.
+
+(Counterargument: a `@font-face` needn’t be declared bold or italic if the underlying glyphs are from a bold or italic variant of the typeface)
+
+ **06.13: CSS Font Loading API**
+
+This is the first metric that elicited an audible “oof” from me. The CSS Font Loading API is currently the only way to group web font repaints. Arguably unnecessary if you’re only using one font file per typeface, which per the implication in in `06.09b` may be common practice!
+
+ **06.14, 06.14b: unicode-range**
+
+This is another metric that I expect was skewed by Google Fonts usage, as Google Fonts uses `unicode-range` in most of (if not all) its CSS. I’d expect this to be less common in userland—but perhaps filtering out Google Fonts requests in the next edition of the Almanac may be possible 😀.
+
+ **06.15: Preconnect**
+
+Wow—less than 2% of pages are using preconnect! Given that Google Fonts is at 75%, this should be higher! Developers: if you use Google Fonts, use preconnect!! Google Fonts, proselytize preconnect more!
+
+**06.16: local()**
+
+See `06.14`, my comments are similar. It has been said (and I agree) that using `local()` can be unpredictable as installed versions of fonts can be outdated and unreliable.
+
+**06.18, 06.21, 06.25: Variable Fonts**
+
+Even at 1.8% this was higher than expected—although I’m excited to see this take off even higher. I would be interested in some deeper analysis here. Is a big player using variable fonts? Can we slice this by host instead of requests?
+
+**06.28 and 06.28b: Variable Font variation axes**
+
+Through the lens of the this large data set, these are very low sample sizes—take these results with a grain of salt. However, `opsz` as the most common axis is notable, with `wght` and `wdth` trailing. In my experience, the introductory demos for variable fonts are usually `font-weight` based.
+
+**06.29: Variable Font variation axes +/- 20pt**
+
+- [ ]  [CONTENT_NEEDED]
+
+**06.30: Variable Font variation axes combinations**
+
+- [ ]  [CONTENT_NEEDED]
+
+**06.26: font-stretch**
+
+`font-stretch` is not a well known `@font-face` descriptor, historically, due to poor browser support. That’s changed and it is now well supported. It has been suggested that using condensed fonts on smaller viewports allows more text to be viewable, but it certainly isn’t common. That being said, that this descriptor is used more on desktop than mobile is unexpected—and ~7% seems much higher than I would have predicted.
```

Comment:
Link to https://developer.mozilla.org/en-US/docs/Web/CSS/font-stretch for people who don't know it